helpers.js

```diff
@@ -3,8 +3,10 @@ export const forumCategoriesObj = {
   409: { category: "Project Feedback", className: "feedback" },
```

Comment:
Could you please change the name of this mapping object `forumCategoriesObj` to something more meaningful like `supportedTopicCategories` and change the property name `category` to `name` for each object?

eg. `409: { name: "Project Feedback", className: "feedback" }`